papertrail/papertrail-services
# Upgrade Ruby and gems

- Upgrade ruby to 2.6.3
- Upgrade a bunch of gems
- Switch test suite to Minitest
- Add a couple of very basic sanity check tests for the sinatra app
- Remove hoptoad
